### ISSUE TITLE
Bug: No card delete confirmation status for some cards

### ISSUE THREAD

**Issue by harshilsharma63** [mattermost/focalboard#4476]
## Steps to reproduce the behavior

1. Create a new card. Add some properties to it and set their values. Add some comments in there as well. Do not set the card title.
2. Go back to kanban view and try deleting the card.
3. Notice the UI didn't show any delete confirmation for this card.
4. Create a new card and repeat step 1 but this time also set a card title.
5. Go back to kanban view and try deleting the card. Notice that this time you see a delete confirmation.

## Expected behavior

Card delete confirmation should show up if there is any data in the card.

## Edition and Platform

 - Edition: Mattermost Boards (plugin)
 - Version: v7.8.0
 - Browser and OS: Firefox on MacOS

## Additional context

The log in webapp is probably if the card has no title and no content, don't show the delete confirmation dialogue. It does not take into consideration the properties and comments in the card.

**Comment by wuwinson** [comment]
Agreed, we should display the deletion confirmation if there is any content on the card - title, properties, comments, card description (content).

**Comment by JesusMurguia** [comment]
I would like to work on this.

**Comment by wuwinson** [comment]
Awesome, thanks @JesusMurguia! Assigned to you. cc @sbishel
